### DISCUSSION TITLE
新版本2.11.4调节亮度反应过慢

### DISCUSSION THREAD

**Discussion by hitopo** [shuax/MouseInc.Settings#91] [Q&A]
滚轮滚动一下是5%的调节，如果滚动过快调节的速度太慢了，之前的版本好像没有这个问题

**Comment by shuax** [comment]
优化了一点点

**Reply by ifermi** [reply]
2.11.5也是，反應很慢。以前的版本是即時顯示，現在是滾動滾輪等5-7秒才能看到調節結果。

**Reply by hitopo** [reply]
我晚上更新了一下，比2.11.4版本好一些，但是就体验来说还是有些延迟

**Comment by hitopo** [comment]
更新了2.11.6版本，现在调整亮度是实时的了，感谢作者

**Reply by ifermi** [reply]
2.11.6亮度調節OK了，大佬辛苦！
